Userland/Libraries/LibJS/AST.h

```diff
@@ -349,18 +349,17 @@ class ScopeNode : public Statement {
 
 // ImportEntry Record, https://tc39.es/ecma262/#table-importentry-record-fields
 struct ImportEntry {
-    DeprecatedFlyString import_name; // [[ImportName]] if a String
-    DeprecatedFlyString local_name;  // [[LocalName]]
-    bool is_namespace { false };     // [[ImportName]] if `namespace-object`
+    Optional<DeprecatedFlyString> import_name; // [[ImportName]] if a String
```

Comment:
Perhaps extend this comment with, present if a String, Empty if namespace-object